### PR TITLE
Sfl 164

### PR DESCRIPTION
**Pull request by saketPandhare-Webonise**

### CODE REVIEW

File: SUNFLOWR/SUNFLOWR/NavigationController/NavigationController.swift [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    
+    @objc func rightBtnPressed(sender: UIButton) {
+        ///Method to be override in subclass
+    }
+    
+    @objc func leftBarButtonPressed(sender: UIButton) {
+       ///Method to be override in subclass
+    }
+    
+
```
Comment (vineet-webonise) [inline SUNFLOWR/SUNFLOWR/NavigationController/NavigationController.swift:77]: Please remove blank space

File: SUNFLOWR/SUNFLOWR/NavigationController/NavigationController.swift [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        navigationItem.leftBarButtonItems = [constructBackBtnForNavBar(imageName: image)]
+    }
+    
+    /// This function is used to set right button for navigation bar
+    /// - Parameter image: image passed from controller
+    func showRightBarButton(image: String) {
+        navigationItem.rightBarButtonItems = [configureRightButtonWithImage(imageName: image)]
+    }
+    
+
```
Comment (vineet-webonise) [inline SUNFLOWR/SUNFLOWR/NavigationController/NavigationController.swift:38]: Please remove blank space

File: SUNFLOWR/SUNFLOWR/NavigationController/NavigationController.swift [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+         setNavigationBarColor()
+    }
+    
+    /// This fucntion is used to set Color for Navigation bar
+    func setNavigationBarColor() {
+           navigationController?.navigationBar.barTintColor = UIColor.gray
+    }
+    
+
```
Comment (vineet-webonise) [inline SUNFLOWR/SUNFLOWR/NavigationController/NavigationController.swift:25]: Please remove blank space

File: SUNFLOWR/SUNFLOWR/NavigationController/NavigationController.swift (hunk 1 above)
Comment (saketPandhare-Webonise) [reply SUNFLOWR/SUNFLOWR/NavigationController/NavigationController.swift]: Done

File: SUNFLOWR/SUNFLOWR/NavigationController/NavigationController.swift (hunk 2 above)
Comment (saketPandhare-Webonise) [reply SUNFLOWR/SUNFLOWR/NavigationController/NavigationController.swift]: Done

File: SUNFLOWR/SUNFLOWR/NavigationController/NavigationController.swift (hunk 3 above)
Comment (saketPandhare-Webonise) [reply SUNFLOWR/SUNFLOWR/NavigationController/NavigationController.swift]: Done

**Pull request merged**